packages/core/src/providers/workos.ts

```diff
@@ -144,15 +144,20 @@ export interface WorkOSProfile extends Record<string, any> {
  * :::
  */
 export default function WorkOS<P extends WorkOSProfile>(
-  options: OAuthUserConfig<P>
+  options: OAuthUserConfig<P> & { connection?: string }
 ): OAuthConfig<P> {
-  const { issuer = "https://api.workos.com/" } = options
+  const {
+    issuer = "https://api.workos.com/",
+    connection = process.env.AUTH_WORKOS_CONNECTION ?? "",
```

Comment:
we cannot use `process.env` in core as it's non-standard. let's just ask the user to pass `connection` for now.

Reply:
Yeah thats what i wrote in the description as well. Sounds good, i'll just change it to take a connection option then :+1:

Reply:
Also the other question I had in the description - for `checks` we had an enum including the option `none`, but not `nonce`. Is `nonce` correct? Should I update the `checks` enum?